src/js/ble_hci_socket_hci.js

```diff
@@ -607,7 +607,7 @@ Hci.prototype.processLeConnComplete = function(status, data) {
   var role = data.readUInt8(2);
   var addressType = data.readUInt8(3) === 0x01 ? 'random': 'public';
   var address = uuidUtil.reverseByteOrder(data.slice(4, 10).toString('hex'), ':');
-  var interval = data.readUInt16LE(10) * 1.25;
+  var interval = data.readUInt16LE(10) * ('1.25' - 0);
```

Comment:
For multiplication you don't need the `- 0`. Only for addition, and '+' sign is faster. E.g.:

`var a = b + +"0.5"`;